Codeinwp/neve
# removing skin mode for hfg rows [closes #1550]

### Summary
Removes the Skin setting for all rows.
Adds new setting for row text color.
Primary menu dropdowns inherit background color / overlay color of the row.


### Will affect visual aspect of the product

Shouldn't - NO

### Test instructions


- Setup the rows on old version, then migrate to the new one. 
- The colors should be properly replaced with no differences.
- Customizer controls defaults/values should correspond to the front end values.
- The migration flag is set under the `neve_migrated_hfg_colors` theme mod so you'll have to delete it if you want to check multiple times. 
- It also fixes an issue where the background control was getting set on customizer initialization for each save.
- Dropdowns should inherit background color of the rows (for primary menu)


Closes #1550.
